src/google.cr

```diff
@@ -689,6 +691,7 @@ class Google::Calendar::Event
       body: @description,
       location: @location,
       attendees: attendees,
+      hide_attendees: hide_attendees,
```

Comment:
This file also needs the place event => Google event
Need to update the `private def event_params(event, calendar_id)` function with reverse mapping